src/vulkan/swap_chain.cpp

```diff
@@ -1,3 +1,4 @@
+#include "ic_log.h"
```

Comment:
C++ general practice is to have the header that corresponds with the cpp file be the top-most include, so ic_log.h would be under swap_chain.h in this instance